doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -659,7 +660,11 @@ Response code:
     + offset: 20 (optional, number) - Establishes the offset from which entities will be returned
     
     + attrs: seatNumber (optional, string) - Comma-separated list of attribute names which data will be included in the response.
-    If this parameter is not included, all the attributes are retrieved.
+    The attributes are retreived in the same order as specified by this parameter. If this parameter is not included, all the attributes
+    are retrieved, in arbitrary order.
```

Comment:
For the entire doc:
in the same order as specified by  => in the order specified by